### PR TITLE
update info for Pangeo Europe weekly meetings

### PR DESCRIPTION
**Pull request by annefou**

### CODE REVIEW

**Review by pl-marasco — APPROVED** [review]
Fine by me, 
tnx

File: docs/meeting-notes.rst [hunk 1]
```diff
@@ -27,16 +27,12 @@ To adress different time zones among the globe continental meetings have been or
 
 1. Europe & Africa
 
-   Gitter: `Pangeo Europe <https://gitter.im/pangeo-data/Europe>`__ 
+   Discord: `Invite to join <https://discord.gg/ex5qqEyyTz>`__ 
 
    - Pangeo European Coffee ; an informal meeting with open discussions:   
       * Schedule: Every week on Tuesday at 9:30am CET
-      * Conferencing: Conferencing: `meet.jit.si/pangeo-europe <https://meet.jit.si/pangeo-europe>`_
+      * Conferencing: Conferencing: `https://numfocus-org.zoom.us/j/81977735338?pwd=pVM3UvnSAJORc2p4Oad39TESPvBzV5.1 <https://numfocus-org.zoom.us/j/81977735338?pwd=pVM3UvnSAJORc2p4Oad39TESPvBzV5.1>`_
```
[the remark this answers is not in the document]
Comment (keewis) [reply docs/meeting-notes.rst]: okay, no, that doesn't appear to have worked. You'll have to use the other change instead (add back the double link text).

[the remark this answers is not in the document]
Comment (annefou) [reply docs/meeting-notes.rst]: I think it works if we only add the link but no quotes.

[the remark this answers is not in the document]
Comment (keewis) [reply docs/meeting-notes.rst]: I agree: the link in the preview appears to show up properly

**Pull request merged**